tests/runtime/test_http_server.py

```diff
@@ -253,9 +253,9 @@ def test_inference_other_task(other_task_model_id):
             f"/api/v1/{other_task_model_id}/task/other",
             json=json_input,
         )
-        assert response.status_code == 200
+        assert response.status_code == 500
```

Comment:
Hm, I don't understand this test. It looks like we took a working test and since the output doesn't have `producer_id` in it, that results in a 500 instead of a 200? If so, I agree with the above comment about needing a better way to test this. Maybe some kind of poison pill in the request that causes a bad response object?

Reply:
Good idea! Added a poison pill